packages/firestore/test/integration/api/fields.test.ts

```diff
@@ -353,33 +354,43 @@ apiDescribe('Timestamp Fields in snapshots', (persistence: boolean) => {
     return { timestamp: ts, nested: { timestamp2: ts } };
   };
 
-  it('are returned as native dates if timestampsInSnapshots set to false', () => {
-    const settings = { ...DEFAULT_SETTINGS };
-    settings['timestampsInSnapshots'] = false;
-
-    const timestamp = new Timestamp(100, 123456789);
-    const testDocs = { a: testDataWithTimestamps(timestamp) };
-    return withTestCollectionSettings(persistence, settings, testDocs, coll => {
-      return coll
-        .doc('a')
-        .get()
-        .then(docSnap => {
-          expect(docSnap.get('timestamp'))
-            .to.be.a('date')
-            .that.deep.equals(timestamp.toDate());
-          expect(docSnap.data()!['timestamp'])
-            .to.be.a('date')
-            .that.deep.equals(timestamp.toDate());
-
-          expect(docSnap.get('nested.timestamp2'))
-            .to.be.a('date')
-            .that.deep.equals(timestamp.toDate());
-          expect(docSnap.data()!['nested']['timestamp2'])
-            .to.be.a('date')
-            .that.deep.equals(timestamp.toDate());
-        });
-    });
-  });
+  // timestampInSnapshots is not support in the modular API.
```

Comment:
Off topic: are we removing this option at Fireconf?

Reply:
I am planning to do so: go/firestore-breaking-changes